config/initializers/app_artifacts.rb

```diff
@@ -1,10 +1,15 @@
 require 'app_artifacts'
 
 AppArtifacts.setup do |store|
+  # SAML keys and certs
   store.add_artifact(:saml_2019_cert, '/%<env>s/saml2019.crt')
   store.add_artifact(:saml_2019_key, '/%<env>s/saml2019.key.enc')
   store.add_artifact(:saml_2020_cert, '/%<env>s/saml2020.crt')
   store.add_artifact(:saml_2020_key, '/%<env>s/saml2020.key.enc')
   store.add_artifact(:saml_2021_cert, '/%<env>s/saml2021.crt')
   store.add_artifact(:saml_2021_key, '/%<env>s/saml2021.key.enc')
+
+  # OIDC keys
+  store.add_artifact(:oidc_private_key, '/%<env>s/oidc.key')
+  store.add_artifact(:oidc_public_key, '/%<env>s/oidc.pub')
```

Comment:
separate random thought, in a future iteration of this DSL we should have it specify which S3 bucket to get the various secrets from, since like the maxmind DB, pwned passwords, etc are in different buckets than this I think

Reply:
I believe everything is in the same secrets bucket except application.yml. The paths for those files are different. They use the `common` prefix instead of the env.